influxdb-logger.groovy

```diff
@@ -582,6 +582,9 @@ def handleEvent(evt, softPolled = false) {
         data += ",unit=${unit} value=${value}"
     }
 
+    // add sample type to the measurement as field
+    data += ",sampleType=\"${sampleType}\""
```

Comment:
I wanted to ask why did you remove your sampleType value from line 394 and add it here. It seems like it should be on 394 as it is part of the information passed with the Event value that is parsed in that block of code.

Reply:
See comments near the top of handleEvent():
`
    // Build data string to send to InfluxDB:
    //  Format: <measurement>[,<tag_name>=<tag_value>] field=<field_value>
    //    If value is an integer, it must have a trailing "i"
    //    If value is a string, it must be enclosed in double quotes.
`
In fact for completeness the format spec should read:

`
    //  Format: <measurement>[,<tag_name>=<tag_value>] field=<field_value> timestamp
`

The difference is that having it up there with deviceName et al. makes it par of the list of tags.   I've put it in with the fields (cf. my hesitations in previous comment).

Reply:
So I guess I am confused as to what you've actually done. If you want to point me to a commit I can look at it, would be easier.